um.cls

```diff
@@ -255,35 +255,37 @@
 
 {\setlength{\parindent}{0em}
 	
-\textbf{(a) Authenticity of Dissertation}\\
+\textbf{(a) Authenticity of Dissertation}\vspace{3mm}
 
 \noindent I hereby declare that I am the legitimate author of this Dissertation and that it is my original work.
 
 No portion of this work has been submitted in support of an application for another degree or qualification of this or any other university or institution of higher education.
 
 I hold the University of Malta harmless against any third party claims with regard to copyright violation, breach of confidentiality, defamation and any other third party right infringement.\\
 
-\textbf{(b) Research Code of Practice and Ethics Review Procedures}\\
+\textbf{(b) Research Code of Practice and Ethics Review Procedures}\vspace{3mm}
 
 I declare that I have abided by the University’s Research Ethics Review Procedures.
 
 As a Master’s student, as per Regulation 58 of the General Regulations for University Postgraduate Awards, I accept that should my dissertation be awarded a Grade A, it will be made publicly available on the University of Malta Institutional Repository.}\\[5mm]
 %%\newcolumntype{L}{>{\arraybackslash}m{2.5in}}
 \bgroup
 \def\arraystretch{1.2}%
-\begin{tabular}{ll}
```

Comment:
Change this line to \begin{tabular}{@{}p{0.4\linewidth}p{0.55\linewidth}} fixes text-warp for long titles